examples/rng-display.rs

```diff
@@ -0,0 +1,106 @@
+//! Generate random numbers using the RNG peripheral and display the values.
+//! This example is specifically tuned to run correctly on the
+//! stm32f4-discovery board (model STM32F407G-DISC1)
+//! This example requires the `rt` feature to be enabled. For example:
+//!
+//! ```bash
+//! cargo run --release --features stm32f407,rt  --example rng-display
+//! ```
+//!
+//! Note that this example requires the `--release` build flag because it is too
+//! large to fit in the default `memory.x` file provided with this crate.
+
+#![no_std]
+#![no_main]
+
+use stm32f4xx_hal as hal;
+
+#[cfg(not(debug_assertions))]
+use panic_halt as _;
+#[cfg(debug_assertions)]
+use panic_semihosting as _;
+
+use cortex_m_rt::ExceptionFrame;
+use cortex_m_rt::{entry, exception};
+use embedded_graphics::{fonts::Font6x8, prelude::*};
+
+use ssd1306::{prelude::*, Builder as SSD1306Builder};
+
+use hal::rng::RngExt;
```

Comment:
Please move this to the prelude so everyone who already uses the prelude can just use it.